internal/resource/nvml-device.go

```diff
@@ -99,3 +100,35 @@ func (d nvmlDevice) GetPCIClass() (uint32, error) {
 	}
 	return nvDevice.Class, nil
 }
+
+func (d nvmlDevice) IsImexCapable() (bool, error) {
+	return true, nil
```

Comment:
How does this behave on older drivers where `GetGpuFabricInfo()` isn't available? Should we implement similar logic to what we have for MIG?

Reply:
the symbol to look for would be `nvmlDeviceGetGpuFabricInfo`

Reply:
done via PR on go-nvlib